common/test/acceptance/pages/lms/video/video.py

```diff
@@ -92,6 +92,7 @@ def wait_for_video_player_render(self, autoplay=False):
         Wait until Video Player Rendered Completely.
 
         """
+        self.wait_for_ajax()
```

Comment:
Why is this required? The next line of code is self.wait_for_video_class(), and the first line of that method is wait_for_ajax().